src/t_zset.c

```diff
@@ -2877,7 +2877,18 @@ struct zrange_result_handler {
 };
 
 /* Result handler methods for responding the ZRANGE to clients. */
-static void zrangeResultBeginClient(zrange_result_handler *handler) {
+static void zrangeResultBeginClient(zrange_result_handler *handler, long hint) {
```

Comment:
let's call it `length` and document that it could be set to -1 if not known in advance.

Reply:
addressed in the latest commit